meta/android-usage-generator.swift

```diff
@@ -10,15 +10,16 @@ Steps:
 */
 
 let data = """
-"Date","13.0","14.0","12.0","11.0","10.0","9.0 Pie","8.0 Oreo","8.1 Oreo","6.0 Marshmallow","7.0 Nougat","5.1 Lollipop","5.0 Lollipop","7.1 Nougat","4.4 KitKat","15.0","4.3 Jelly Bean","24.1","4.2 Jelly Bean","4.1 Jelly Bean","23.1","24.7","4.0 Ice Cream Sandwich","Other"
-2024-11,18.74,36.45,13.11,11.9,6.25,3.73,2.91,1.27,1.9,0.92,0.46,1.39,0.28,0.13,0.41,0.02,0.05,0.01,0.01,0,0.01,0.01,0.05
+"Date","14.0","13.0","12.0","11.0","15.0","10.0","9.0 Pie","8.0 Oreo","6.0 Marshmallow","5.0 Lollipop","8.1 Oreo","7.0 Nougat","5.1 Lollipop","7.1 Nougat","16.0","4.4 KitKat","4.3 Jelly Bean","24.1","24.7","25.2","4.1 Jelly Bean","Other"
```

Comment:
Something is strange here, why is 15 in the middle of the list? And why is 16 not listed?